src/cairo_run.rs

```diff
@@ -13,3 +16,39 @@ pub fn cairo_run(path: &str) {
     cairo_runner.relocate().unwrap();
     cairo_runner.write_output(&mut io::stdout());
 }
+
+#[allow(dead_code)]
```

Comment:
Why do you need dead code?

Reply:
resolved in [dbd01e9](https://github.com/lambdaclass/cleopatra_cairo/pull/144/commits/dbd01e917b47e6c3ce0e6ded300ce1277cb0693a)